mesh/confirmation/views.py

```diff
@@ -0,0 +1,182 @@
+from django.http import HttpResponse
+from django.core.mail import send_mail
+from django.template.loader import render_to_string
+from django.utils.html import strip_tags
+from ..accounts.models import Account
+from ..accountSettings.models import Settings
+from ..profiles.models import Profile
+import secrets
+import os
+import time
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Generate random token for user, store it, and send confirmation email
+# which then leads into the confirm_token function
+def email_confirmation(request, user_email):
+    # Grab account ID by user_email
+    account_id = get_account_id(user_email)
+
+    # Check if an account was found
+    if account_id is None:
+        return HttpResponse(user_email + ": User does not exist!")
+    
+    # Grab account email verification status
+    email_already_verified = get_verification_status(account_id)
+
+    # Check if user email is verified yet or not
+    # if so, skip generating new token
+    if email_already_verified:
+        return HttpResponse(user_email + ": User email already verified!")
+
+    # Prevent user from generating too many tokens/receiving too many emails
+    settings_data = get_settings_data(account_id)
+    
+    seconds_since_last_token = calculate_timestamp_difference(settings_data["verificationToken"])
+    minutes_since_last_token = seconds_since_last_token // 60
+    
+    if seconds_since_last_token <= 600:
```

Comment:
Use constants! Replace 600 with something like `SECONDS_COOLDOWN = 600`. Try to avoid having floating numbers in your code if it's meaningful.

Reply:
Good point! Will fix.